src/Microsoft.AspNetCore.Dispatcher/Resources.resx

```diff
@@ -121,4 +121,46 @@
     <value>Multiple endpoints matched. The following endpoints matched the request:{0}{0}{1}</value>
     <comment>0 is the newline - 1 is a newline separate list of action display names</comment>
   </data>
+  <data name="TemplateRoute_CannotHaveCatchAllInMultiSegment" xml:space="preserve">
```

Comment:
A note to our future selves, we will need to review and polish these names and strings when we get further along.